lazyflow/operators/ioOperators/opInputDataReader.py

```diff
@@ -73,7 +73,7 @@ class OpInputDataReader(Operator):
     videoExts = ['ufmf', 'mmf', 'avi']
     h5Exts = ['h5', 'hdf5', 'ilp']
     klbExts = ['klb']
-    npyExts = ['npy']
+    npyExts = ['npy', 'npz']
```

Comment:
In addition to this change, you should consider actually setting the `InternalPath` in `OpInputDataReader._attemptOpenAsNpy()`, below.

We can assume the same conventions we use for hdf5.  That is, the `FilePath` value can be parsed as `/path/to/file.npy/internalName`.  We have a little utility class named `PathComponents` defined in `lazyflow.utility` that can help with this. (See `lazyflow/utility/pathHelpers.py`.)  You'll have to make a slight modification to the `PathComponents` source code, since currently it only allows files with an hdf5 extension to have an `internalPath`.

Once you do all that, then ilastik's headless mode will support `.npz` files with user-specified internal paths.  Supporting them in the GUI, too, would require yet more work, and probably isn't worth the trouble right now.